e2e/tests/childSolicitors_test.js

```diff
@@ -128,8 +182,46 @@ function assertChild(I, idx, child, solicitor) {
       I.seeInTab([childElement, 'Representative', 'Organisation (unregistered)', 'Organisation address', 'Postcode/Zipcode'], solicitor.unregisteredOrganisation.address.postcode);
       I.seeInTab([childElement, 'Representative', 'Organisation (unregistered)', 'Organisation address', 'Country'], solicitor.unregisteredOrganisation.address.country);
     } else {
-      I.waitForText(solicitor.organisation, 40);
-      I.seeOrganisationInTab([childElement, 'Representative', 'Name'], solicitor.organisation);
+      I.waitForText(solicitor.details.organisation, 40);
+      I.seeOrganisationInTab([childElement, 'Representative', 'Name'], solicitor.details.organisation);
     }
   }
 }
+
+async function attemptAndFailNoticeOfChange(I, noticeOfChangePage, solicitor, child) {
+  await I.signIn(solicitor);
+  await noticeOfChangePage.userFillsNoC(caseId, 'Swansea City Council', child.value.party.firstName, child.value.party.lastName);
+  I.click('Continue');
+  I.see('Enter the client details exactly as they’re written on the case, including any mistakes');
```

Comment:
Again, instead of using `I.see(...)` which does an immediate check, it might be more robust to either `I.waitForText(...)` for that line or `I.waitForElement(...)` to wait for an attribute of the error before running the check.